locales/core/de.json

```diff
@@ -0,0 +1,208 @@
+    "notification_plural": "Benachrichtigungen",
+    "wentWrong": "Etwas ist falsch gelaufen! Bitte versuche es erneut"
+  },
+  "ok": "Okay",
+  "oneMoment": "Einen Moment bitte...",
+  "optional": "Optional",
+  "partyShuffle": "Party Shuffle",
+  "partyShuffleTooltip": "Spiele einen Mix aus all deinen Favoriten in einer übersichtlichen Playlist ab",
+  "passwordReset": "Passwort zurücksetzen",
+  "passwordResetSuccess": "Erfolgreich! Ändere jetzt dein Passwort und du bist fertig!",
+  "pastHour": "Letzte Stunde",
+  "pinned": "Angepinnt",
+  "plan": "Plan",
+  "playNext": "Als Nächstes abspielen",
+  "playSimilar": "Ähnliche Songs abspielen",
+  "playlist": "Playlist",
+  "playlistDelete": "Aus deiner Playlist löschen",
+  "playlistDeleteCreator": "Diese Playlist wurde vom Ersteller gelöscht.",
+  "playlist_plural": "Playlists",
+  "preferences": "Einstellungen",
+  "radio": "Radio",
+  "redirecting": "Leite um...",
+  "related": "Ähnlich wie",
+  "repeat": {
+    "all": "Alle wiederholen",
+    "one": "Einen Song wiederholen",
+    "repeat": "Wiederholen"
+  },
+  "restoreDefault": "Standard wiederherstellen",
+  "rising": "Aufsteigend",
+  "save": "SPEICHERN",
+  "saveAmount": "Speichere '{{amount}}'",
+  "saveTunes": "\"Save some tunes!\"",
+  "saved": "GESPEICHERT",
+  "search": "Suche",
+  "searchCheckout": "Suche nach neuen Songs oder probiere eine neue Radio-Station aus",
+  "selectColor": "Farbschema auswählen",
+  "selectPlan": "Plan auswählen",
+  "shuffle": "Shuffle",
+  "signOut": "Ausloggen",
+  "signUp": "Registrieren",
+  "signUpNow": "Jetzt Registrieren",
+  "station": {
+    "cannotLoad": "Kann {{stationKey}} nicht laden",
+    "random": "Zufällige Station wählen",
+    "station_plural": "Stationen"
+  },
+  "stepWithCount": "Schritt {{count}}",
+  "stepWithCount_plural": "Schritt {{count}}",
+  "stickBasics": "Nein, danke, ich bleibe beim Basic-Plan.",
+  "theme": "Theme",
+  "thisMonth": "Dieser Monat",
+  "thisWeek": "Diese Woche",
+  "thisYear": "Dieses Jahr",
+  "title": "Titel",
+  "today": "Heute",
+  "top": "Top",
+  "track": {
+    "addCollection": "Titel zu '{{name}}' hinzugefügt",
+    "addQueue": "Titel zur Warteschleife hinzugefügt",
+    "noneFound": "Keine Titel gefunden",
+    "removeCollection": "Titel von '{{name}}' entfernt",
+    "removeFromQueue": "Von Warteschleife entfernen",
+    "removedQueue": "Von Warteschleife entfernt",
+    "track_plural": "Titel",
+    "withCount": "{{count}} Titel",
+    "withCount_plural": "{{count}} Titel"
+  },
+  "tryFilter": "Vielleicht einen anderen Filter ausprobieren?",
+  "tryPro": "Probiere <1>Pro</1> 30 Tage gratis!",
+  "unlike": "Like entfernen",
+  "updatePassword": "Passwort ändern",
+  "upgrade": "Upgraden",
+  "upgradeFor": "Upgrade zu <1>Pro</1> für $1.99/Monat und halte Headset am Leben! Genieße alle Premium-Features und habe ein gutes Gefühl dabei, jemanden zu helfen!",
+  "upgradeNow": "Jetzt upgraden",
+  "upgradePro": "__NOT_TRANSLATED__",
```

Comment:
I've added this new key that needs translating now

Reply:
Done